client/src/components/DemoButton/DemoButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import login from '../../helpers/APICalls/login';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { CircularProgress } from '@material-ui/core';
+
+const demoUser = { email: 'user@demo.com', password: 'password' };
```

Comment:
I think this feature was already a part of a previous PR which has been approved? 🤔 

When you have some time to spare, I would highly recommend creating a new repo and practice creating PRs - branching off of other branches - rebasing branches - et cetera.